### PR TITLE
Edits to Pgd/5/consistency/crdt/docs 287 PR5928

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/pgd/5/consistency/crdt/01_crdt_usage.mdx [hunk 1]
```diff
@@ -64,21 +66,21 @@ SELECT bdr.alter_table_conflict_detection('crdt_example',
...
+Suppose you now issue the same SQL on two nodes at same time as you did in the non-CRDT example: 
 
 ```sql
 UPDATE crdt_example
    SET counter = counter + 1    -- "reflexive" update
  WHERE id = 1;
 ```
 
-Once the changes are applied, you find that the counter has managed to concurrenct updates.
+Once the changes are applied, you find that the counter has managed two concurrent updates:
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/consistency/crdt/01_crdt_usage.mdx:83]: Please verify this edit. The correction to concurrent is correct but I wanted to verify that to was meant to be two. Otherwise, a verb is missing.

File: product_docs/docs/pgd/5/consistency/crdt/01_crdt_usage.mdx [hunk 2]
```diff
@@ -138,10 +140,9 @@ SELECT id, counter FROM crdt_delta_example WHERE id = 1;
 (1 row)
 ```
 
-With a regular `integer` column, the result is `2`. But when you update the row with a delta CRDT counter, you start with the OLD row version, make a NEW row version, and send both to the remote node. There, compare them with the version found there (e.g., the LOCAL version). Standard CRDTs merge the NEW and the LOCAL version, while delta CRDTs compare the OLD and NEW versions and apply the delta
+With a regular `integer` column, the result is `2`. But when you update the row with a delta CRDT counter, you start with the OLD row version, make a NEW row version, and send both to the remote node. There, compare them with the version found there (for example, the LOCAL version). Standard CRDTs merge the NEW and the LOCAL version. Delta CRDTs compare the OLD and NEW versions and apply the delta
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/consistency/crdt/01_crdt_usage.mdx:143]: Are the u/c words here being used for emphasis or is this referring to something appears in code? If emphasis, we don't use all caps for this purpose or really any kind of emphasis, and I would lower case them. If referring to something that appears in the code, that's fine.

File: product_docs/docs/pgd/5/consistency/crdt/02_state-op-crdts.mdx [hunk 3]
```diff
@@ -20,22 +20,26 @@ The main disadvantage is that you can't reset this value reliably in an asynchro
 
 !!! Note
     Implementing more complicated operation-based types by creating custom data types is possible, storing the state and the last operation. (Every change is decoded and transferred, so multiple
-    operations aren't needed). But at that point, the main benefits (simplicity, reuse of existing data types) are lost without gaining any advantage compared to state-based types (for example, still no capability to reset) except for the space requirements. (A per-node state isn't needed.)
+    operations aren't needed.) But at that point, the main benefits (simplicity, reuse of existing data types) are lost and no advantage gained compared to state-based types except for the space requirements. (A per-node state isn't needed.) For example, there's still no ability to reset.
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/consistency/crdt/02_state-op-crdts.mdx:23]: Please check this edit.I was trying to break up a long sentence and too many parentheticals, and I wasn't 100% sure I got this right.

File: product_docs/docs/pgd/5/consistency/crdt/05_crdt-reset.mdx [hunk 4]
```diff
@@ -7,7 +7,7 @@ Resetting CRDT values is possible but requires special handling. The asynchronou
 cluster means that different nodes might see the reset operation at different places in the change stream no matter how it's implemented. Different nodes might also initiate a reset concurrently, that is, before
 observing the reset from the other node.
 
-In other words, to make the reset operation behave correctly, it needs to be commutative with respect to the regular operations. Many naive ways to reset a value that might work well on a single-node fail
+In other words, for the reset operation to behave correctly, it needs to be commutative with respect to the regular operations. Many naive ways to reset a value that might work well on a single-node fail
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/consistency/crdt/05_crdt-reset.mdx:10]: Wanted to verify the use of "naive" as it's not something I've seen before. Is this standard vocabulary? Or was "native" meant?

**Comment by ebgitelman** [comment]
@jpe442 Overall this looked really great editorially. Most significant edits were for long sentences and overuse of parentheses. In some cases, commas would do just as well. In others, they made the sentence hard to follow. Generally, parentheticals reduce readability. 

I did have a handful of queries, mostly to verify that my edit was correct.

File: product_docs/docs/pgd/5/consistency/crdt/05_crdt-reset.mdx (hunk 4 above)
Comment (ebgitelman) [reply product_docs/docs/pgd/5/consistency/crdt/05_crdt-reset.mdx:10]: @djw-m Would "basic" work?

**Comment by ebgitelman** [comment]
@jpe442 Please review so I can get this merged. Thank you!

File: product_docs/docs/pgd/5/consistency/crdt/01_crdt_usage.mdx (hunk 1 above)
Comment (jpe442) [reply product_docs/docs/pgd/5/consistency/crdt/01_crdt_usage.mdx:83]: This does look right. Nice catch!

File: product_docs/docs/pgd/5/consistency/crdt/05_crdt-reset.mdx (hunk 4 above)
Comment (jpe442) [reply product_docs/docs/pgd/5/consistency/crdt/05_crdt-reset.mdx:10]: My suggestion would be "straightforward" or "obvious".

File: product_docs/docs/pgd/5/consistency/crdt/02_state-op-crdts.mdx (hunk 3 above)
Comment (jpe442) [reply product_docs/docs/pgd/5/consistency/crdt/02_state-op-crdts.mdx:23]: Are you sure? If you take out the phrase "the main benefits...are lost", the sentence becomes "But at that point no advantage gained compared to state-based types except for the space requirements..." I believe the way "and" is functioning there it requires the "is" to make the sentence valid. ChatGPT agrees and says it is the only way to ensure subject/verb agreement.

File: product_docs/docs/pgd/5/consistency/crdt/01_crdt_usage.mdx (hunk 2 above)
Comment (ebgitelman) [reply product_docs/docs/pgd/5/consistency/crdt/01_crdt_usage.mdx:143]: Okay, will leave.

File: product_docs/docs/pgd/5/consistency/crdt/02_state-op-crdts.mdx (hunk 3 above)
Comment (ebgitelman) [inline product_docs/docs/pgd/5/consistency/crdt/02_state-op-crdts.mdx:23]:
```suggestion
    operations aren't needed.) But at that point, the main benefits (simplicity, reuse of existing data types) are lost and no advantage is gained compared to state-based types except for the space requirements. (A per-node state isn't needed.) For example, there's still no ability to reset. 
```

Comment (ebgitelman) [reply product_docs/docs/pgd/5/consistency/crdt/02_state-op-crdts.mdx:23]: I think it's easier to read with the verb, so I added "is."

**Comment by ebgitelman** [comment]
@djw-m @piano35-edb @jpe442 Thanks for the useful feedback. This should be ready to approve now. Thanks!

**Review by djw-m — APPROVED** [review]
LGTM (but will it merge?)
